scripts/check-versions.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+archs="amd64 arm64 s390x"
+
+for release in $(cat versions.txt); do
+  for arch in $archs; do
+    RESP=$(curl -L -o /dev/null --silent -Iw '%{http_code}' "https://dl.k8s.io/release/${release}/bin/linux/${arch}/kubectl")
+    if [ "$RESP" != "200" ]; then
```

Comment:
Could you simply use `curl --fail` instead of checking for 200? `--fail` makes curl exit with an error if the response is not a 2xx code.

```
$ curl -f https://sbstp.ca/foo.txt
curl: (22) The requested URL returned error: 404
```

If you combine that with `set -e` it will exit the script on the first error.